src/tiled/commanddatamodel.h

```diff
@@ -25,6 +25,7 @@
 #include <QAbstractTableModel>
 #include <QSettings>
 
+class QKeySequence;
```

Comment:
Hmm, I didn't notice this earlier, but you can't actually rely on a forward-declaration for a class that is used as a return value. The reason it compiles anyway is because `command.h` is already including the QKeySequence header. I'd say just get rid of this line.